scripts/configure-ironic.sh

```diff
@@ -118,54 +113,16 @@ if [ -n "${HTTP_BASIC_HTPASSWD}" ]; then
     fi
 fi
 
-
-# When running both API and Conductor in the same container, we'll try to get the credentials
-# from /auth/ironic-rpc/auth-config if present, or generate it
-if [ "${IRONIC_DEPLOYMENT}" == "Combined" ]; then
-    # We try to read the credentials from the config file as it is probably mounted read-only,
-    # We cannot modify it. If it is not set to basic, then do not authenticate the RPC. This is
-    # to ensure that the setup will work if the user gives a specific config for rpc set to no_auth
-    if [ -f "/auth/ironic-rpc/auth-config" ]; then
-        IRONIC_RPC_TMP_TYPE="$(crudini --get /auth/ironic-rpc/auth-config json_rpc auth_type)" || exit 1
-        if [ "${IRONIC_RPC_TMP_TYPE}" == "http_basic" ]; then
-            IRONIC_RPC_TMP_USERNAME="$(crudini --get /auth/ironic-rpc/auth-config json_rpc username)" || exit 1
-            IRONIC_RPC_TMP_PASSWORD="$(crudini --get /auth/ironic-rpc/auth-config json_rpc password)" || exit 1
-        else
-            export JSON_RPC_AUTH_STRATEGY="noauth"
-        fi
-    # We do not have an auth config file, so we generate one
-    else
-        IRONIC_RPC_TMP_USERNAME="rpc-user"
-        IRONIC_RPC_TMP_PASSWORD="$(tr -dc 'a-zA-Z0-9' < /dev/urandom | fold -w 12 | head -n 1)"
-        mkdir -p "/auth/ironic-rpc"
-        cat << EOF > "/auth/ironic-rpc/auth-config"
-[json_rpc]
-auth_type=http_basic
-username=${IRONIC_RPC_TMP_USERNAME}
-password=${IRONIC_RPC_TMP_PASSWORD}
-http_basic_username=${IRONIC_RPC_TMP_USERNAME}
-http_basic_password=${IRONIC_RPC_TMP_PASSWORD}
-EOF
-    fi
-
-    # Populate HTTP_BASIC_HTPASSWD_RPC
-    if [ -n "${IRONIC_RPC_TMP_USERNAME:-}" ]; then
-        htpasswd -n -b -B "${IRONIC_RPC_TMP_USERNAME}" "${IRONIC_RPC_TMP_PASSWORD}" >"${HTPASSWD_FILE}-rpc"
-    fi
-fi
-
-. /bin/coreos-ipa-common.sh
```

Comment:
Was this removed on purpose?

Reply:
mmmm no

Reply:
In that case, lets keep it. This was present in the PR by @dtantsur to enable support for ignition embedding in the downstream fork. Thanks.